txpool/mock_test.go

```diff
@@ -13,7 +13,9 @@ type defaultMockStore struct {
 }
 
 func (m defaultMockStore) Header() *types.Header {
-	return &types.Header{}
+	return &types.Header{
+		GasLimit: 4712388,
```

Comment:
@zivkovicmilos 
Is it possible to add a method to the defaultMockStore for setting the gas limit of the latest header?
Right now only 1 test uses this value I think, but in the future more unit tests may rely on this GasLimit value to be configurable

Reply:
Not directly, because defaultMockStore is mapped to store object, and we can call only mock methods from that interface. There are a few ways we can bypass this. One idea comes to my mind.
1. create a constructor for initialization defaultMockStore object. Which can receive header as attribute
2. when initializing the pool, we can send header data with what we want to initialize

Reply:
Can we go with solution 1?

I don't feel good about hardcoding these values, even if it's a mock store. Someone will need to untangle it in some other PR if we leave it like this

Reply:
This is one possible solution. It is a bit redundant with sending nil value where we want a default value. But go does not like optional parameters. 55eeead6ab4c2431394e0611e1e6969516e6f74f

Reply:
@brkomir @lazartravica Here is the discussion why i had to add header parameter